plugins/modules/elb_application_lb_info.py

```diff
@@ -69,77 +69,154 @@
+            type: int
             sample: 60
         ip_address_type:
-            description:  The type of IP addresses used by the subnets for the load balancer.
+            description: The type of IP addresses used by the subnets for the load balancer.
             type: str
-            sample: ipv4
+            sample: "ipv4"
+        listeners:
+            description: Information about the listeners.
+            type: complex
+            contains:
+                listener_arn:
+                    description: The Amazon Resource Name (ARN) of the listener.
+                    type: str
+                    sample: ""
+                load_balancer_arn:
+                    description: The Amazon Resource Name (ARN) of the load balancer.
+                    type: str
+                    sample: ""
+                port:
+                    description: The port on which the load balancer is listening.
+                    type: int
+                    sample: 80
+                protocol:
+                    description: The protocol for connections from clients to the load balancer.
+                    type: str
+                    sample: "HTTPS"
+                certificates:
+                    description: The SSL server certificate.
+                    type: complex
+                    contains:
+                        certificate_arn:
+                            description: The Amazon Resource Name (ARN) of the certificate.
+                            type: str
+                            sample: ""
+                ssl_policy:
+                    description: The security policy that defines which ciphers and protocols are supported.
+                    type: str
+                    sample: ""
+                default_actions:
+                    description: The default actions for the listener.
+                    type: str
+                    contains:
+                        type:
+                            description: The type of action.
+                            type: str
+                            sample: ""
+                        target_group_arn:
+                            description: The Amazon Resource Name (ARN) of the target group.
+                            type: str
+                            sample: ""
         load_balancer_arn:
             description: The Amazon Resource Name (ARN) of the load balancer.
             type: str
-            sample: arn:aws:elasticloadbalancing:ap-southeast-2:0123456789:loadbalancer/app/my-elb/001122334455
+            sample: "arn:aws:elasticloadbalancing:ap-southeast-2:0123456789:loadbalancer/app/my-elb/001122334455"
         load_balancer_name:
             description: The name of the load balancer.
             type: str
-            sample: my-elb
+            sample: "my-elb"
+        routing_http2_enabled:
+            description: Indicates whether HTTP/2 is enabled.
+            type: bool
+            sample: true
```

Comment:
Same as on the earlier comment from @alinabuzachis probably no point including this here